### PR TITLE
Little Update

### PR DESCRIPTION
**Pull request by KingTheRipper**
## Short Description:
Adding a little modify and one new file
## Fixes/Resolves/Closes (please use correct syntax):
- Add a full list of Pokémon for a better autorelease, with any leve for each
- Change simple setting in config.json `"auth_service": "google or ptc"` to become simpler fro a new users choose account type
- Add a Pokeballs types in to items filters (have to set the values according to your LV)

### CODE REVIEW

File: configs/config.json.example [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "auth_service": "google",
+    "auth_service": "google or ptc",
```
Comment (simonsmh) [inline configs/config.json.example]: This would broken setup.sh.

Comment (KingTheRipper) [inline configs/config.json.example]: changed with: 

```
"//auth_service choose google or ptc (Pokemon Trainer Club)",
"auth_service": "google",
```

**Comment by SGColdSun** [comment]
The ptc line in your config is incorrectly tpc.

**Comment by k4n30** [comment]
@KingTheRipper please rebase on lastest dev, then fix as persgcoldsuns comments, and fix travis errors (if any)

**Comment by solderzzc** [comment]
Code should be submit to dev branch

**Comment by mjmadsen** [comment]
Wrong branch. Please create PR to dev branch.

**Pull request closed without merging**